tools/validators/instance_validator/main.py

```diff
@@ -38,6 +40,11 @@
                       required=True,
                       help='Filepath to YAML building configuration',
                       metavar='FILE')
+  parser.add_argument('-m', '--modified-ontology-types',
```

Comment:
please use FLAGS as in here: https://github.com/google/digitalbuildings/blob/master/tools/rdf_generator/rdfformat/rdf_generator.py#L28

Reply:
Will refactor later